tests/python/relay/test_pass_partial_eval.py

```diff
@@ -311,7 +311,16 @@ def test_concat():
     x = Var("x", t)
     y = Var("x", t)
     orig = run_infer_type(Function([x, y], op.concatenate([x, y], axis=0)))
-    assert_alpha_equal(orig, dcpe(orig))
+    assert_alpha_equal(dcpe(orig), orig)
+
+
+def test_triangle():
+    t = relay.TensorType([], "int32")
+    x = Var("x", t)
+    f_var = Var("f")
+    f = Function([x], If(op.equal(x, const(0)), const(0), x + f_var(x - const(1))))
+    orig = run_infer_type(Let(f_var, f, f_var(const(10))))
+    assert_alpha_equal(dcpe(orig), const(55))
```

Comment:
Why is this called `test_triangle`?

Reply:
triangle number

Reply:
can you change it to `test_triangular_number`?